core/src/main/java/org/apache/accumulo/core/client/impl/Namespaces.java

```diff
@@ -72,10 +78,10 @@ public String invalidMessage(String namespace) {
     }
   };
 
-  public static final String DEFAULT_NAMESPACE_ID = "+default";
-  public static final String DEFAULT_NAMESPACE = "";
-  public static final String ACCUMULO_NAMESPACE_ID = "+accumulo";
-  public static final String ACCUMULO_NAMESPACE = "accumulo";
+  public static final Namespace.ID DEFAULT_NAMESPACE_ID = Namespace.ID.DEFAULT;
+  public static final String DEFAULT_NAMESPACE = Namespace.DEFAULT;
+  public static final Namespace.ID ACCUMULO_NAMESPACE_ID = Namespace.ID.ACCUMULO;
+  public static final String ACCUMULO_NAMESPACE = Namespace.ACCUMULO;
```

Comment:
Agreed.  This was an attempt to minimize number of changes in one PR.  Assuming there aren't any showstoppers with these changes, I will create follow on ticket.